devtools/gradle/pom.xml

```diff
@@ -10,7 +10,7 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>quarkus-gradle-plugin</artifactId>
+    <artifactId>io.quarkus.gradle.plugin</artifactId>
```

Comment:
do we absolutely need to change this ? 

if we do - doesn't gradle have some capability of handling renames somewhat transparently ?

Reply:
See https://github.com/quarkusio/quarkus/pull/5761#discussion_r354286243. This was corrected to the artifactId that is understood by Gradle, allowing us to remove the `resolutionStrategy` snippet in settings.gradle